tests/test_tokenization.py

```diff
@@ -0,0 +1,89 @@
+import sys
+import os
+
+SCRIPT_DIR = os.path.dirname(os.path.abspath(__file__))
+sys.path.append(os.path.dirname(SCRIPT_DIR))
+
+from gector.tokenization import *
+import unittest
+from transformers import AutoTokenizer
+
+class TokenizationTests(unittest.TestCase):
```

Comment:
No comments at all. For someone who's generally unfamiliar with GECToR, it would be useful to have some context of what the test is supposed to be testing exactly.